docs/api-reference/userdefinednetwork-api-spec.md

```diff
@@ -9,6 +9,8 @@
 Package v1 contains API Schema definitions for the network v1 API group
 
 ### Resource Types
+- [ClusterUserDefinedNetwork](#clusteruserdefinednetwork)
```

Comment:
@ormergi : Let's do a new md file for CUDNs? I'd prefer a file per resource - this file already seems long enough will lots of sections
WDYT?

Reply:
Currently both types lives under the same directory pkg/crd/userdefinednetwork.
Having additional file for CUDN will require some manual changes because the generator we use cannot generate content for single file.

Reply:
@ormergi ack then let's not complicate it, I am fine with them being on same page but can you please update the main reference page: https://ovn-kubernetes.io/api-reference/introduction/#ovn-kubernetes-features-api-reference add ClusterUDNs and then link that to the subsection of cluster udns ?

Reply:
@ormergi ^ ?

Reply:
I tried that but it didn't work, it seem mkdocs doesnt support setting the site navigation (nad in mkdocs.yaml) to have two different titles points to the same file, and point nav title to markdown title (that is the CUDN title in UDN doc).
At least not out of the box, maybe its possible with some python/JS hacks and hooks, but it will require more time to realize how to do it.

I created an issue asking how it can be done https://github.com/mkdocs/mkdocs/issues/3869

Reply:
thanks @ormergi if you feel its too much time sink its fine we can drop this idea, but I wanted to somehow highlight there are two CRD's so maybe we can forget the hyperlink for now and leave the text

I see they responded with https://github.com/mkdocs/mkdocs/issues/3428#issuecomment-2061111631 and don't plan to fix this in mkdocs, don't want to waste too much of your time

Reply:
Yeah I am pretty clueless about how to workaround the limitation mkdocs has.

FWIW, first thing you see when clicking the UserDefinedNetwork tab is the types UDN and CUDN.

Reply:
yeah let's just get this in.